internal/cmd/config/profile/unset/unset.go

```diff
@@ -31,6 +32,14 @@ func NewCmd(p *print.Printer) *cobra.Command {
 				return fmt.Errorf("unset profile: %w", err)
 			}
 
+			flow, err := auth.GetAuthFlow()
+			if err != nil {
+				p.Debug(print.WarningLevel, "both keyring and text file storage failed to find a valid authentication flow for the active profile")
+				p.Warn("Failed to find a valid authentication flow for the active profile. Please login using the 'stackit auth login' command.\n")
```

Comment:
The unset command always sets the profile to the default one right? We could also make it more clear in this message that the **default** profile is not authenticated